cmd/tt/bucket_util.go

```diff
@@ -0,0 +1,199 @@
+package main
```

Comment:
Lots of cruft needed to leverage the local bucket diffing in the `bucket` command